index.js

```diff
@@ -239,7 +239,7 @@ class StaticComponentEnveloper {
     const publicPath = this.getPublicPath({ compilation })
 
     return chunks.reduce((assets, chunk) => {
-      const chunkFiles = chunk.files.map((chunkFile) => {
+      const chunkFiles = [...chunk.files].map((chunkFile) => {
```

Comment:
I think you'll get a `TypeError` using the spread operator on a non-iterable (like null or undefined). But maybe it's not worth overthinking it! This way at least we make sure we're operating on a plain ole' JS array. Yay for no docs.

Reply:
You are absolutely right about the TypeError. I'll revers it to the array concat one.